From f09f6dc2c8f59b2b58159cc413b605a547c8646e Mon Sep 17 00:00:00 2001
From: Andy Chi <andy.chi@canonical.com>
Date: Tue, 29 Mar 2022 15:36:13 +0800
Subject: [PATCH] hwdb: Add mic mute key mapping for HP Elite x360

On the new Elite x360 2 in 1 HP laptops, the microphone mute hotkey is "Fn+F8" and
the scancode for this hotkey is 0x81, but this scancode was mapped to
fn_esc in the HP generic keymap section. To fix this problem, we add
a machine specific keymap section to add the correct keymap rule.
---
 hwdb.d/60-keyboard.hwdb | 2 ++
 1 file changed, 2 insertions(+)

Index: systemd-ubuntu-core/hwdb.d/60-keyboard.hwdb
===================================================================
--- systemd-ubuntu-core.orig/hwdb.d/60-keyboard.hwdb
+++ systemd-ubuntu-core/hwdb.d/60-keyboard.hwdb
@@ -621,6 +621,8 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett
 # HP EliteBook
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPEliteBook*:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPEliteBook*:*
+# HP Elite x360
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPElite*x360*:*
 # HP Elite Dragonfly
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPEliteDragonfly*:*
 # HP ProBook 440 G2
